### PR TITLE
BDRK-3245: Add securityContext variables

### PR DESCRIPTION
**Pull request by tunguyen9889**
- Add securityContext variables for pods and containers.
- Update tfline and Terraform version in test Dockerfile.
- Update latest Chart version and Ambassador image tag (see [Ambassador Chart](https://artifacthub.io/packages/helm/datawire/ambassador)).
- Add CODEOWNERS.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -161,14 +161,26 @@ variable "env_raw" {
   default     = []
 }
 
+variable "pod_security_context" {
+  description = "Pod securityContext"
+  type        = map(string)
```
Comment (lawliet89) [inline variables.tf:166]: Same thing.

```suggestion
  type        = map(any)
```

File: variables.tf [hunk 2]
```diff
@@ -161,14 +161,26 @@ variable "env_raw" {
...
+  default     = {}
+}
+
+variable "container_security_context" {
+  description = "Container securityContext"
+  type        = map(string)
```
Comment (lawliet89) [inline variables.tf:172]:
```suggestion
  type        = map(any)
```

File: README.md [hunk 3]
```diff
@@ -67,6 +67,7 @@ the individual modules.
 | <a name="input_certificates"></a> [certificates](#input\_certificates) | ARN of certificates in ACM to use | `list(string)` | `[]` | no |
 | <a name="input_chart_namespace"></a> [chart\_namespace](#input\_chart\_namespace) | Namespace to run the chart in | `string` | `"ambassador"` | no |
 | <a name="input_chart_version"></a> [chart\_version](#input\_chart\_version) | Version of Chart to install. Set to empty to install the latest version | `string` | `"6.6.3"` | no |
+| <a name="input_container_security_context"></a> [container\_security\_context](#input\_container\_security\_context) | Container securityContext | `map` | `{}` | no |
```
Comment (lawliet89) [inline README.md:70]: BTW, this is generated from https://github.com/terraform-docs/terraform-docs

Comment (tunguyen9889) [reply README.md:70]: Yes, I will take a look at the template and update it if any.

**Review by lawliet89 — APPROVED** [review]

**Pull request merged**